tests/installation/leap2sle_zdup.pm

```diff
@@ -34,7 +34,6 @@ sub run {
 
     systemctl 'set-default --force multi-user.target';
     fully_patch_system();
-    zypper_call 'in SUSEConnect';
```

Comment:
That part might still be necessary, at least on <= 15.3

Reply:
changed && thanks